app/models/manageiq/providers/ansible_tower/configuration_manager/configuration_script/api_create.rb

```diff
@@ -0,0 +1,30 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationScript
+  module ApiCreate
+    def create_in_provider(manager_id, params)
+      manager = ExtManagementSystem.find(manager_id)
+      job_template = manager.with_provider_connection do |connection|
+        connection.api.job_templates.create(params)
+      end
+
+      manager.refresh_ems #Get the record in our database
+
+      find_by(:manager_id => manager.id, :manager_ref => job_template.id)
+    end
+
+    def create_in_provider_queue(manager_id, params)
+      task_opts = {
+        :action => "Creating Ansible Tower Job Template",
+        :userid => "system"
+      }
+
+      queue_opts = {
```

Comment:
@gtanzillo / @jrafanie  This needs more options to target the correct worker (but I don't know what).  Can you help with that?